demisto_sdk/commands/common/tools.py

```diff
@@ -2650,3 +2652,12 @@ def extract_none_deprecated_command_names_from_yml(yml_data: dict) -> list:
         if command.get('name') and not command.get('deprecated'):
             commands_ls.append(command.get('name'))
     return commands_ls
+
+
+def remove_copy_and_dev_suffixes_from_str(field_name: str) -> str:
+    split_by_delimiter = len(field_name.split('_'))
+    for index in range(split_by_delimiter - 1):
```

Comment:
```suggestion
    for _ in range(field_name.count('_') - 1):
```

the `-1` may not be necessary

Reply:
done